pinot-core/src/test/java/org/apache/pinot/core/util/TableConfigUtilsTest.java

```diff
@@ -544,55 +545,43 @@ public void testValidateIndexingConfig() {
     }
 
     // Although this config makes no sense, it should pass the validation phase
-    StarTreeIndexConfig starTreeIndexConfig = new StarTreeIndexConfig(Arrays.asList("myCol"),
-        Arrays.asList("myCol"),
-        Arrays.asList("SUM__myCol"),
-        1);
+    StarTreeIndexConfig starTreeIndexConfig =
+        new StarTreeIndexConfig(Arrays.asList("myCol"), Arrays.asList("myCol"), Arrays.asList("SUM__myCol"), 1);
     tableConfig = new TableConfigBuilder(TableType.OFFLINE).setTableName(TABLE_NAME)
-        .setStarTreeIndexConfigs(Arrays.asList(starTreeIndexConfig))
-        .build();
+        .setStarTreeIndexConfigs(Arrays.asList(starTreeIndexConfig)).build();
     try {
       TableConfigUtils.validate(tableConfig, schema);
     } catch (Exception e) {
       Assert.fail("Should fail for valid StarTreeIndex config column name");
```

Comment:
Unrelated to this PR, but is this supposed to be "should not fail" and also not `//expected` ?

Reply:
You're right. Lemme update that.